client_plugin/Makefile

```diff
@@ -281,6 +288,29 @@ CLEANESX_SH   := $(DEPLOY_TOOLS) cleanesx
 deploy-esx: clean-esx
 	$(DEPLOY_ESX_SH) "$(ESX)" "$(VIB_BIN)"
 
+test-upgrade:
+	$(BUILD) upgrade-test
+upgrade-test:
```

Comment:
test-upgrade: and upgrade-test will leads to confusion 

1. better to have inline comment
2. `upgrade-tes` needs to be renamed.